RFC/src/RFC-0201_TariScript.md

```diff
@@ -507,12 +506,28 @@ For Bob to claim his commitment, \\( C_b \\) he requires the blinding factor, \\
 
 In this case, the script hash is analogous to an address in Bitcoin or Monero. Bob's wallet can scan the blockchain
 looking for hashes that he would know how to resolve. For all outputs that he discovers this way, Bob would need to know
-who the sender is so that he can derive the shared secret.  This information can be obtained without needing to communicate
-with the sender, if she uses the offset public key, \\( K_{Ob} \\) as the key in deriving the Diffie-Hellman exchange.
+who the sender is so that he can derive the shared secret.  
+
+This information can be obtained without needing to communicate with the sender. If the blinding factor \\( k_b \\) was calculated with diffie-hellman using the offset public keypair, (\\( k_{Ob} \\),\\( K_{Ob} \\)) as sender keypair and
+the keypair, (\\( k_{Sb} \\),\\( K_{Sb} \\)) as the receiver keypair, the blinding factor \\( k_b \\) can be securely calculated without communication between the 2 parties.
+
+Alice calculates \\( k_b \\) as
+$$
+    k_b = k_{Ob} * {K_Sb}
+$$
+
+Bob calculates \\( k_b \\) as
+$$
+    k_b = K_{Ob} * {k_Sb}
+$$
 
 For Bob's part, when he discovers one-sided payments to himself, he should spend them to new outputs using a traditional
 transaction to thwart any potential horizon attacks in the future.
 
+One other problem with one sided communication is the communication of the value of the transaction being sent, and the wallet recovery of one sided payments. We can solve both of these by using (\\( k_{Ob} \\).
```

Comment:
```suggestion
One other problem with one sided communication is the communication of the value of the transaction being sent, and the wallet recovery of one sided payments. We can solve both of these by using (\\( k_b \\).
```

Reply:
We should be using k_{Ob}, the offset pubkey. k_{b} is the blinding factor.

Reply:
Bob does not know `k_{Ob}` though?

Reply:
Yes, He only knows `K_{Ob}` and `k_S`

Reply:
The wording is ambiguous, but the next line is correct, so will let it pass